style.css

```diff
@@ -144,160 +155,91 @@
   font-weight: 600px;
   font-size: 17px;
   color: white;
   border-radius: 4px;
-  width: 120px;
-  height: 47px;
+  width: 122px;
+  height: 48px;
   border: 2px solid #36b37f;
+  transition: background-color 1s;
 }
 
-.button:hover {
-  transform: rotate(360deg);
-  transform: scale(1.01);
-}
-
-.button:active {
+.get-button:active {
   background: #008552;
   border-radius: 4px;
 }
 
-.About-me {
-  background-color: #fff;
-  padding-top: 40px;
-  padding-bottom: 40px;
-  background-image: url(./Images/illustration-\ about\ me-\ mobile12.png);
-  background-repeat: no-repeat;
-  background-position: center top;
-}
-
-.descr {
-  display: flex;
-  flex-direction: row;
-  justify-content: flex-end;
-}
-
-.resume {
-  display: flex;
-  flex-direction: row;
-  justify-content: left;
+.get-button:hover {
+  background-color: #d0d9d4;
+  box-shadow: 0 8px 16px rgba(54, 179, 127, 0.24);
 }
 
-.about-me {
-  width: 375px;
-  height: auto;
+.about_me_container {
   display: flex;
-  padding: auto 24px;
-  overflow: hidden;
-  position: relative;
-  align-items: center;
-  flex-shrink: 0;
-  border-color: transparent;
   flex-direction: column;
-}
-
-.aboutme-text {
-  display: flex;
-  overflow: hidden;
-  position: relative;
+  align-content: center;
   align-items: center;
-  border-color: transparent;
-  margin-bottom: 58px;
-  flex-direction: column;
-  text-align: center;
-  font-family: 'Inter', sans-serif;
-}
-
-.about-title {
-  color: #172b4d;
-  width: 351px;
-  height: 48px;
-  font-size: 24px;
-  align-self: auto;
-  font-style: bold;
-  font-family: 'Inter', sans-serif;
+  justify-content: space-between;
+  flex-wrap: wrap;
   text-align: center;
-  font-weight: 700;
-  text-decoration: none;
-  line-height: 52px;
+  margin: 68px 12px 24px 12px;
+  background-image: url("Mask-Group-top.png"), url("Frame.png");
+  background-position: right top, left bottom;
+  background-repeat: no-repeat;
 }
 
-.about-text1 {
-  color: #172b4d;
-  width: 327px;
-  height: 363;
-  font-size: 20px;
-  padding-bottom: 10px;
-  text-align: center;
-  font-weight: 400;
+.about_me1 {
+  margin-top: 40px;
+  padding: 6% 10% 6% 10%;
 }
 
-.aboutme-button {
-  display: flex;
-  padding: 12px;
-  overflow: hidden;
-  position: relative;
-  align-items: center;
-  border-color: transparent;
-  border-radius: 4px;
-  background-color: #36b37e;
-  color: #fff;
-  text-align: center;
+.about_me2 {
+  padding: 6% 10% 6% 10%;
 }
 
-.aboutme-button:hover {
-  background-color: #36b37e;
-  transition: 0.7s;
+.resume_button {
+  font-family: "Inter", sans-serif;
+  background-color: #36b37f;
+  font-weight: 500px;
+  font-size: 20px;
+  color: white;
+  border-radius: 5px;
+  padding: 10px 15px;
+  border: 2px solid #36b37f;
+  margin: 20px 10px 20px 10px;
+  transition: background-color 1s;
 }
 
-.aboutme-button:active {
-  background-color: #36b37f;
+.resume_button:hover {
+  background-color: #d0d9d4;
+  transform: rotate(360deg);
+  transform: scale(1.01);
 }
```

Comment:
Please consider adding the functionality of `hover and press` on the button. So when the button is hovered or pressed its style changes as follows. Currently, your styles are not in accordance with the design. To achieve this you can add the following functionality of code into your CSS file.

## Solution:--

```
.classname:hover{
    background-color: #36B37F;
    color: white;
}
```

```
.classname:active{
    background-color: #008552;
    color: white;
}
```

## Demonstration:--

![image](https://user-images.githubusercontent.com/94223367/222924036-ec7c1353-54de-484a-b446-68be07b7ae09.png)